package.json

```diff
@@ -4,54 +4,65 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "format": "prettier --write \"{components,content,docs,pages,styles,utils}/**/*.{js,jsx,ts,tsx,css,scss,json,yaml,md}\"",
-    "format:check": "prettier --check \"{components,content,docs,pages,styles,utils}/**/*.{js,jsx,ts,tsx,css,scss,json,yaml,md}\"",
-    "lint": "eslint --fix '{components,pages,utils}/**/*.{js,jsx,ts,tsx}'",
-    "lint:check": "eslint '{components,pages,utils}/**/*.{js,jsx,ts,tsx}'",
-    "prepare": "husky install",
-    "pre-commit": "npm run format && git add -A ."
+    "lint": "eslint '{components,pages,utils}/**/*.{js,json,jsx,ts,tsx}'",
+    "lint:fix": "yarn lint --fix",
+    "commit": "commit",
+    "prepare": "husky install"
   },
   "dependencies": {
-    "@octokit/core": "^3.5.1",
+    "@next/font": "^13.1.1",
```

Comment:
Could you please explain more about it?

Reply:
About removing octokit? It was not being used. @next/font? This is the way you import fonts in nextjs for performance. We were using classic css imports previously